ci/terraform/oidc/site.tf

```diff
@@ -37,7 +37,12 @@ provider "aws" {
 
   default_tags {
     tags = {
-      environment = var.environment
+      Environment = var.environment
+      Owner       = "di-orchestration@digital.cabinet-office.gov.uk"
```

Comment:
Is the owner `di-orchestration` or is it auth as the frontend-api is deployed via this terraform (and not sure if orchs lambdas are still deployed by terraform)?

Reply:
entirely reasonable. I'll switch this out.